kolibri/core/webpack/hooks.py

```diff
@@ -521,7 +521,7 @@ class FrontEndCoreHook(WebpackInclusionHook):
 
     def __init__(self, *args, **kwargs):
         super(FrontEndCoreHook, self).__init__(*args, **kwargs)
-        assert len(list(self.registered_hooks)) <= 1, "Only one core asset allowed"
```

Comment:
`self.registered_hooks` is a generator, so it doesn't have a length, hence it being coerced to a list first.